anabot/runtime/installation/hub/connect_to_redhat/__init__.py

```diff
@@ -152,6 +152,38 @@ def password_check(element, app_node, local_node):
     # Password input is the first password in the credentials panel
     return aptie(getnode(local_node, 'password text'), value, 'Account')
 
+def combo_str(value):
+    if value is None:
+        return tr('Not Specified')
+    return value
+
+# This defines combo box that appears for user in multiple organizations
+def account_organization_combo_handler(element, app_node, local_node):
+    org = combo_str(get_attr(element, 'value'))
+    org_combo = getnodes(local_node, 'combo box')[0]
+    if org_combo.name != org:
+        org_combo.click()
+        combo_window = getnode(app_node, "window")
+        getnode(combo_window, "menu item", org).click()
+    return PASS
+
+def account_organization_combo_check(element, app_node, local_node):
+    value = combo_str(get_attr(element, 'value'))
+    combobox = getnodes(local_node, 'combo box')[0]
+    if combobox.name != value:
+        return Fail('Different value was present: "%s". Expected: "%s".' % (combobox.name, value))
+    return PASS
+
+@handle_act('/account_organization')
+def account_organization_handler(element, app_node, local_node):
```

Comment:
I don't see a need for a definition of the `account_organization_handler` and `account_organization_check` functions as the only thing they do is calling `account_organization_combo_handler` and `account_organization_combo_check` respectively. The thing that can be done here is to not have the `account_organization_handler` and `account_organization_check` functions at all and just decorate the `account_organization_combo_handler` and `account_organization_combo_check` respectively.